### DISCUSSION TITLE
Preview Deployment Pro Plan not working

### DISCUSSION THREAD

**Discussion by gangataarun** [vercel/community#2509] [Help] [answered]
### Summary

Hi,

I recently upgraded to the pro plan (free trial) on Vercel for my Next.js project. However, I'm encountering an issue with the preview deployment not being triggered when I make commits to the preview branch. Strangely, in the free version of my project, the preview deployment works correctly after commits to the preview branch.

My application is currently in the testing phase, and I decided to switch to the pro plan because I was experiencing function timeouts on the free plan. I have verified that Vercel has the necessary access to my Git repository and all branches. I am deploying the project using the same method in both versions, using the UI interface (GitHub commits using GH Desktop and Next.js on the Vercel UI).

I'm unsure why the behavior of the preview deployment is different between the pro plan and the free plan. Any insights or suggestions to resolve this issue would be greatly appreciated.

Thank you!

**Comment by amyegan** [comment]
Preview deployments should work the same between Hobby and Pro accounts. It sounds like the project might have the [`ignoreCommand`](https://vercel.com/docs/concepts/projects/project-configuration#ignorecommand) in vercel.json or [Ignored Build Step](https://vercel.com/docs/concepts/projects/overview#ignored-build-step) in Project Settings configured to skip Preview builds. Or possibly [`git.deploymentEnabled`](https://vercel.com/docs/concepts/projects/project-configuration/git-configuration#git.deploymentenabled) set to `false` for the preview branch.

**Reply by gangataarun** [reply]
I dont have any Ignored Build Step configured. im deploying using the UI so I dont have a vercel.json file. I have checked the project settings and i don't see any settings applied

**Comment by dennis-bonfire** [comment, marked as answer]
I just had the same issue. After transferring from Hobby to Pro, Vercel had deleted my custom domain (even though the Vercel docs say domains are transferred).

I added back my custom domain for the preview environment, and that resolved my issue.

**Comment by phojie** [comment]
I SHOULD HAVE NOT UPGRADE TO PRO PLAN, THIS SERVICE IS SO ANNOYING

**Reply by amyegan** [reply]
Hi @phojie. You don't need to upgrade to a Pro plan unless your project is commercial or exceeds the Hobby tier included usage. If you have any specific feedback about your experience, please let me know so I can share it with the rest of the team 🙏 

https://vercel.com/docs/limits/fair-use-guidelines
